shopinvader_sale_cart_anonymous_partner/tests/test_promotion.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Akretion (http://www.akretion.com).
+# @author Florian Mounier <florian.mounier@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from unittest import mock
+
+from odoo.tests.common import TransactionCase
+
+from odoo.addons.shopinvader_anonymous_partner.models.res_partner import COOKIE_NAME
+
+
+class PartnerPromotionCase(TransactionCase):
```

Comment:
Please use `BaseCommon` class to disable mail tracking

Reply:
wdym?